.github/workflows/test-go1.10.yaml

```diff
@@ -22,6 +22,14 @@ jobs:
         # NOTE: checkout the code in a fixed location, even for forks, as this
         # is relevant for go's import system.
         path: ./src/github.com/snapcore/snapd
+
+    - name: Cache snapd test results
+      id: cache-snapd-test-results
+      uses: actions/cache@v1
+      with:
+        path: {{ github.workspace }}/.test-results
+        key: {{ github.run_id }}-{{ github.job }}-code-{{ hashFiles('**/*') }}
```

Comment:
I think hashFiles is unnecessary, if I read the docs right `run_id` is sufficient (along with job)